cli/internal/cmd/miniup.go

```diff
@@ -142,16 +143,18 @@ func (m *miniUpCmd) prepareConfig(cmd *cobra.Command, fileHandler file.Handler,
 			return nil, errors.New("not overwriting existing config")
 		}
 	}
-
-	config := config.Default()
-	config.Name = constants.MiniConstellationUID
-	config.RemoveProviderAndAttestationExcept(cloudprovider.QEMU)
-	config.StateDiskSizeGB = 8
-
+	if featureset.CurrentEdition != featureset.EditionEnterprise { // TODO: or !featureset.CanFetchMeasurements? image is also needed..
```

Comment:
The measurements for the image should already be embedded in the CLI.
Checking for the edition should be enough.
Alternatively you could add another feature flag, e.g. `CanMiniUp` if you want to explicitly control access to this command.

Reply:
If we want this nice message, I agree that it should use a separate feature `CanMiniUp` or `CanMini`

Reply:
I don't like `canMini` because it implies that the command doesn't work, when it really only means that it doesn't work without config flag. Why is Enterprise not good enough? I think it won't change that generating a complete config is only supported on Enterprise?

Reply:
I would like to remove the actual edition from the features if possible.
How about: `CanUseEmbeddedMeasurements` or something similar? That sounds like we may be able to reuse it in other places in a meaningful way.

Reply:
agree with not coupling edition and a specific feature. changing it